features/admin.sms-templates.v1/pages/sms-customization.tsx

```diff
@@ -0,0 +1,457 @@
+                            />
+                        </Segment>
+                    </Segment.Group>
+
+                    <Segment padded={ true }>
+                        <Show
+                            when={ featureConfig?.smsTemplates?.scopes?.update }
+                        >
+                            {
+                                (!isTemplateNotAvailable || hasSmsTemplateCreatePermissions) && (
+                                    <SmsCustomizationFooter
+                                        isSaveButtonLoading={ isSmsTemplatesListLoading || isSmsTemplateLoading }
+                                        onSaveButtonClick={ handleSubmit }
+                                    />
+                                )
+                            }
+                        </Show>
+                    </Segment>
+                </Segment.Group>
+
+                <Divider hidden/>
+
+                { smsTemplate && !isSystemTemplate && selectedLocale !== I18nConstants.DEFAULT_FALLBACK_LANGUAGE &&
+                (<Show
+                    when={ featureConfig?.smsTemplates?.scopes?.delete }
+                >
+                    <DangerZoneGroup sectionHeader={ t("common:dangerZone") }>
+                        <DangerZone
+                            data-componentid={ `${ componentId }-remove-sms-provider-config` }
+                            actionTitle={ t("extensions:develop.smsTemplates.dangerZone.remove.action") }
+                            header={ t("extensions:develop.smsTemplates.dangerZone.remove.heading") }
+                            subheader={ t("extensions:develop.smsTemplates.dangerZone.remove.message") }
+                            onActionClick={ handleDeleteRequest }
+                        />
+                    </DangerZoneGroup>
+                </Show>)
+                }
+
+                { smsTemplate && !isSystemTemplate && selectedLocale === I18nConstants.DEFAULT_FALLBACK_LANGUAGE &&
+                (<Show
+                    when={ featureConfig?.smsTemplates?.scopes?.delete }
+                >
+                    <DangerZoneGroup sectionHeader={ t("common:dangerZone") }>
+                        <DangerZone
+                            data-componentid={ `${ componentId }-revert-sms-provider-config` }
+                            actionTitle={ t("extensions:develop.smsTemplates.dangerZone.revert.action") }
+                            header={ t("extensions:develop.smsTemplates.dangerZone.revert.heading") }
+                            subheader={ t("extensions:develop.smsTemplates.dangerZone.revert.message") }
+                            onActionClick={ handleDeleteRequest }
+                        />
+                    </DangerZoneGroup>
+                </Show>)
+                }
+            </PageLayout>
+        </BrandingPreferenceProvider>
+    );
+};
+
+/**
+ * Default props for the component.
+ */
+SmsCustomizationPage.defaultProps = {
```

Comment:
Let's not use defaultProps in new components as defaultprops are being deprecated.